apps/cookbook/src/app/localization/localization.component.html

```diff
@@ -0,0 +1,97 @@
+<article>
+  <h1>Localization</h1>
+  <p>
+    Locale information is used by components such as
+    <code>kirby-calendar</code>
+    and
+    <code>input type="date"</code>
+    to automatically format values according to locale. Support has been added for three locales out
+    of the box: American English (
+    <code>en-US</code>
+    ), British English (
+    <code>en-GB</code>
+    ) and Danish (
+    <code>da</code>
+    ).
+  </p>
+  <p>
+    Specifying the locale will also ensure translation of
+    <em>internal</em>
+    component labels and texts that are not configurable. This allows assistive technology such as
+    screen readers to announce labels in the correct language.
+  </p>
+
+  <p>
+    In this example visible information such as the month and weekday headers, and accessible names
+    for headers, calendar cells and interactive buttons have been automatically translated by
+    specifying the locale.
+  </p>
+
+  <cookbook-example-viewer>
+    <div class="example-frame">
+      <div class="locale-examples">
+        <cookbook-da-locale-provider style="position: relative">
+          <kirby-card>
+            <kirby-calendar [selectedDate]="selectedDate"></kirby-calendar>
+          </kirby-card>
+          <kirby-avatar
+            class="locale-avatar"
+            size="sm"
+            imageSrc="/assets/images/dk.svg"
+            altText="Danish locale"
+          ></kirby-avatar>
+        </cookbook-da-locale-provider>
+        <cookbook-en-locale-provider style="position: relative">
+          <kirby-card>
+            <kirby-calendar [selectedDate]="selectedDate"></kirby-calendar>
+          </kirby-card>
+          <kirby-avatar
+            class="locale-avatar"
+            size="sm"
+            imageSrc="/assets/images/gb.svg"
+            altText="English locale"
+          ></kirby-avatar>
+        </cookbook-en-locale-provider>
+      </div>
+    </div>
+  </cookbook-example-viewer>
+
+  <h2>Using the service directly</h2>
+  <p>
+    If custom components are closely coupled to Kirby and need to use identical translations,
```

Comment:
Maybe improve readability:
```suggestion
    When developing custom components closely coupled to Kirby that need to use identical translations,
```